shared/domains/chelonia/chelonia.js

```diff
@@ -567,6 +565,26 @@ export default (sbp('sbp/selectors/register', {
     // calling this will make pubsub unsubscribe for events on `contractID`
     sbp('okTurtles.events/emit', CONTRACTS_MODIFIED, state.contracts)
   },
+  'chelonia/contract/disconnect': async function (contractID, contractIDToDisconnect) {
```

Comment:
Is there a corresponding `'chelonia/contract/connect'`? If not, why not? It seems like there should be one for clarity.

Reply:
There isn't by that name, but it'd be `/out/keyRequest'.